src/services/datahub-client.ts

```diff
@@ -106,6 +106,7 @@ export class HttpDatahubClient implements DatahubClient {
       `domains/${domainName}/events/summarized-by-page`,
       {
         startDate: dateFrom.toISOString(),
+        sortBy: 'visitors',
```

Comment:
![image](https://user-images.githubusercontent.com/1157864/60460950-89b68100-9c1b-11e9-94fa-def5af07fcc7.png)

I think that _totalVisits_ name is wrong now, because this value is not the _total of visits_ anymore, now it is the _count of visitors_ or _unique visits_, Am I missing something?

Reply:
fixed